src/Core/AdminConsole/Enums/OrganizationStatusType.cs

```diff
@@ -3,5 +3,6 @@
 public enum OrganizationStatusType : byte
 {
     Pending = 0,
-    Created = 1
+    Created = 1,
+    Managed = 2,
```

Comment:
This is set when an organization is fully brought under an MSP in Consolidated Billing.